src/main/java/gregtech/api/mui/GTGuiTextures.java

```diff
@@ -268,6 +270,19 @@ public static class IDs {
             .canApplyTheme()
             .build();
 
+    public static final UITexture MC_BUTTON = new UITexture.Builder()
+            .location("modularui", "gui/widgets/mc_button.png") // todo
```

Comment:
What needs addressing for this `todo` and the one for the disabled button?

Reply:
Technically could be nothing, but I left the todo since its using a texture file from MUI that we might want to just copy over to GT for stability/later modification